.github/workflows/ci.yml

```diff
@@ -192,15 +192,14 @@ jobs:
         run: ${{ matrix.execute }}
       - name: ${{ matrix.test_name }} Perfomance Test
         id: perfomance-test
+        timeout-minutes: 5
         run: |
           set -xe
 
           # We need to increase the log level to make sure that they don't hold off storm of packets
           # generated by UDP tests. Wire is especially chatty.
           sed -i 's/^\(\s*\)RUST_LOG:.*$/\1RUST_LOG: wire=error,info/' docker-compose.yml
           cat docker-compose.yml | grep RUST_LOG
-          docker compose down client gateway relay
```

Comment:
If these are not in a valid state after performing the test above, wouldn't the previous step have caught it?

Reply:
We need to take everything down so that the configuration change we did above with grep is applied

Reply:
Yeah `up` should recreate, which should be a tad faster than `down` and `up`